Makefile

```
@@ -0,0 +1,11 @@
+.PHONY: setup up down
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)